locale/en/site.json

```diff
@@ -77,8 +77,8 @@
         "link": "docs",
         "text": "Docs",
         "es6": {
-            "link": "docs/es6",
-            "text": "ES6 in Node.js"
+            "link": "http://node.green",
+            "text": "ES6 and beyond"
```

Comment:
ES6 is also a valid name. ES2015 is a bit long and the tab doesn't look as good.